### DISCUSSION TITLE
Missing the icon to create  a .dsektop file

### DISCUSSION THREAD

**Discussion by ConstantMan** [timschneeb/GalaxyBudsClient#163] [Q&A] [answered]
As the title say i just want  to create a desktop file in order to quickly find this amazing app ,but i can't figure out where is the icon in order to add the path to it in galaxy-buds.desktop

**Comment by timschneeb** [comment, marked as answer]
The icon is available here: 
PNG (Large) https://github.com/ThePBone/GalaxyBudsClient/raw/master/GalaxyBudsClient/Resources/icon_white.png
PNG (24x24) https://raw.githubusercontent.com/ThePBone/GalaxyBudsClient/master/GalaxyBudsClient/Resources/icon_white_small.png

**Reply by ConstantMan** [reply]
Thanks i downloaded and added it to the desktop file but unfortunately when i launch the app i get one icon from the galaxy-buds.desktop and one from the actual application, anyway to solve this?

**Reply by whale124** [reply]
Can I see your .desktop file?
Why do you want to apply two icons?
